automations/js/src/project_thread_updates.js

```diff
@@ -61,6 +61,7 @@ const GET_PROJECT_CARDS = `
  */
 module.exports = async ({ github, core }) => {
   try {
+    // eslint-disable-next-line no-constant-binary-expression
     const isDryRun = process.env.DRY_RUN === "true" ?? false
```

Comment:
This one caught my eye. Isn't `?? false` redundant here? It doesn't seem to have any effect on the value of the expression.

<img width="311" alt="Screenshot 2024-10-31 at 8 51 01 AM" src="https://github.com/user-attachments/assets/58bc53f7-1167-4d76-b84a-20532248b20e">


```suggestion
    // eslint-disable-next-line no-constant-binary-expression
    const isDryRun = process.env.DRY_RUN === "true" ?? false
```